### ISSUE TITLE
some notebooks not updated to collection 3

### ISSUE THREAD

**Issue by Kooie-cate** [GeoscienceAustralia/dea-notebooks#916]
A number of notebooks in the repo still load landsat collection 2 dataset. For example a number of the Beginners_Guide notebooks including 04_Loading_Data and 05_Plotting load collection 2 Geomedians.

**Comment by robbibt** [comment]
Running this:
```
import glob
pattern = '*/*.ipynb'
query = '_geomedian_annual'

for filepath in glob.iglob(pattern, recursive=True):
    with open(filepath) as file:
        s = file.read()
        if (s.find(query) > -1):
            print(filepath)
```
Indicates these notebooks need to be updated:
```
Beginners_guide/04_Loading_data.ipynb
Beginners_guide/09_Parallel_processing_with_Dask.ipynb
Beginners_guide/05_Plotting.ipynb
Beginners_guide/03_Products_and_measurements.ipynb
Frequently_used_code/Exporting_NetCDFs.ipynb
Frequently_used_code/Machine_learning_with_ODC.ipynb
```

**Comment by robbibt** [comment]
I think this has been addressed in #917 so closing for now - let's re-open if we notice any other notebooks that are using C2 data!
